src/main/java/step3/domain/Lotto.java

```diff
@@ -0,0 +1,66 @@
+package step3.domain;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Objects;
+
+public class Lotto {
+
+    private static final int LOTTO_NUMBER_COUNT = 6;
+
+    private final List<LottoNumber> numbers;
+
+    public Lotto(Collection<LottoNumber> numbers) {
+        validationSize(numbers);
```

Comment:
자바 컨벤션상 일반적으로 메소드 네이밍은 동사 또는 동사구로 시작하는것을 권장 드려요. 😄
```suggestion
        validateSize(numbers);
```

다른곳들도 같이 체크 부탁드립니다 😊